docs/TermsAndConditionsForTheScrum.md

```diff
@@ -1,55 +1,168 @@
+|            2 | Very simple to do. The acceptance criteria are short and can be satisfied with ease | Low                                    | None                                                                    | Little effort needed                                                                 |
+|            3 | Simple to do. The acceptance criteria are longer but clear and manageable.          | Low                                    | Little. Can still be done. May need to consult peers                    | It takes some time to complete                                                       |
+|            5 | Doable, with a few aspects that may be difficult or long.                           | Medium - might need mitigation plan    | Little. Can still be done. May need to consult peers or other resources | Requires especial focus to complete                                                  |
+|            8 | Doable but difficult or long                                                        | Medium - should have a mitigation plan | Medium. Pair programming is likely.                                     | It takes significant amount of sprint to complete this story                         |
+|           13 | Hard. Difficult or complicated. There is a lot of work needed to satisfy this task. | High - must have a mitigation plan     | Medium. Task is on the border of needing a spike to investigate it      | Full time effort likely to take the whole sprint to complete                         |
+|           21 | Impossible for a sprint. Task is too big, break into smaller tasks                  | Extreme - Should not be in a sprint    | High. Team has no idea. Create a spike                                  | The story takes significant effort and will require more than one sprint to complete |
+
+# Issue life cycle
+
+New issues are [triaged](#triage) by the team. This activity is driven by the
+[Special Interest Groups (SIGs)](../community/sig-list.md) and contributors, and
+coordinated in the [sprint meetings](#sprint).
+
+When the team considers that an issue is well understood (i.e. has [good quality](#quality)) and it fits within the scope of the project, the issue gets labeled as `triage/accepted`. If there are missing details that prevent that, a `triage/needs-information` is added.
+
+When the assignee starts to work on an issue, the issue is labeled `lifecycle/active`.
+
+<a name="requirements"></a>
+## Requirements for issues to be planned
+
+An issue must meet the following criteria before it can be planned:
+
+- have an assignee
+- be `triage/accepted`
+- have a defined priority that is aligned with the sprint goals
+
+Feature requests (`type/feature`) must also have:
```

Comment:
type or kind?

Reply:
good catch, thanks. It is `kind/feature` indeed